kie-server-parent/kie-server-services/kie-server-services-common/pom.xml

```diff
@@ -196,6 +234,69 @@
       <artifactId>kie-soup-maven-support</artifactId>
     </dependency>
 
+  	<!-- fabric8 kubernetes and openshift java client dependencies -->
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model</artifactId>
+    </dependency>
+	<dependency>
+		<groupId>io.fabric8</groupId>
+		<artifactId>kubernetes-client</artifactId>
+	</dependency>
+	<dependency>
+		<groupId>io.fabric8</groupId>
+		<artifactId>openshift-client</artifactId>
+	</dependency>
+	<dependency>
+		<groupId>io.fabric8</groupId>
+		<artifactId>kubernetes-server-mock</artifactId>
+	</dependency>
+	<dependency>
+		<groupId>io.fabric8</groupId>
+		<artifactId>openshift-server-mock</artifactId>
+	</dependency>
+
+    <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
```

Comment:
If this is needed by tests please add "test" scope for this dependency so it won't be transitively propagated to dependent modules.

Reply:
Updated POM as per suggested.